src/emailcontactstorage.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Open Mobile Platform LLC.
```

Comment:
I guess this should be `2020 - 2021` now

Reply:
done